charts/telepresence/templates/licenseSecret.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if and .Values.licenseKey.create .Values.licenseKey.secret.create }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.licenseKey.secret.name }}
+  labels:
+    {{- include "telepresence.labels" . | nindent 4 }}
+data:
+  hostDomain: {{ printf "auth.datawire.io" | b64enc | quote }}
+  license: {{ .Values.licenseKey.value | quote | b64enc | quote }}
```

Comment:
I think what you want is `{{ .Values.licenseKey.value | b64enc | quote }}`. I compared a helm template with the secret that telepresence generated and it wasn't giving me the correct value bc it was adding the quotes to the secret.

Reply:
Good catch. I fixed the issue that caused me to need that by just adding the "" to the values file